<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Moving</string>
    </property>
    <widget class="QDoubleSpinBox" name="entry_dx">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="entry_dy">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="button_move">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Move</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dx">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>49</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>dx:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dy">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>49</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>dy:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>950</width>
      <height>750</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Scaling</string>
    </property>
    <widget class="QDoubleSpinBox" name="entry_kx">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="entry_ky">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="button_scale">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_kx">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>kx:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ky">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>49</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ky:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>710</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Rotating</string>
    </property>
    <widget class="QDoubleSpinBox" name="entry_alpha">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="button_rotate">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_alpha">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>95</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Alpha :</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
